src/content/development/release-process/_index.en.md

```diff
@@ -65,16 +65,15 @@ for the job listed but the important one is `Release Images`. When complete, the
 green check mark on success or a red X on failure. A failure likely means the artifacts were not published to Quay, in
 which case select the failed check, inspect the logs, correct the issue and re-run the job.
 
-### Step 0: Create a stable branch (if necessary)
+### Sometimes: Create stable branches for stable releases
 
-If you're creating a new stable release, the release automation process can create the stable branches. To do so,
-navigate to the [releases](https://github.com/submariner-io/releases) repository and fork it.
+If you're creating release that will receive backports, a stable release not a milestone release, you need to create stable branches for
+each repository. The release automation process can do this for you. To do so, navigate to the
```

Comment:
```suggestion
If you're creating release that will receive backports and is not a milestone release, you need to create stable branches for
each repository. The release automation process can do this for you. To do so, navigate to the
```

Reply:
I'm not sure I understand the goal here. If it's just to clarify, it's not more clear to me because it seems to imply there are milestone releases that will receive backports, which I don't think is the case.

Reply:
yeah. I was just trying to clarify the phrase "_a stable release not a milestone release_" - it seems misplaced. How about "_If you're creating release that will receive backports, eg a milestone release, you need to..._".

Reply:
Humm, that's the inverse of what I'm saying and mean to say though. We backports only to sable releases, not milestone releases. I'm trying to say if you're creating a stable release, not a milestone release, you need a branch for backports.

Reply:
I split it up into multiple sentences, maybe that's more clear.